twenty-first/Cargo.toml

```diff
@@ -29,7 +29,8 @@ features = ["precommit-hook", "run-cargo-clippy", "run-cargo-fmt"]
 [dependencies]
 anyhow = "1.0"
 bincode = "1.3"
-bfieldcodec_derive = "0.2.0"
+# bfieldcodec_derive = "0.2.0"
+bfieldcodec_derive = { path = "../bfieldcodec_derive" }
```

Comment:
This does not seem like it belongs in a commit, does it?

Reply:
Good point. We should release a new version of `bfieldcodec_derive` and depend on that in `twenty-first`, and potentially release a new version of `twenty-first` after that.